components/table/test/table.vdiff.js

```diff
@@ -27,6 +27,19 @@ function createSortableHeaderRow() {
 		</tr>
 	`;
 }
+
+function createSortableHeaderRowWithDoubleButton() {
```

Comment:
Another case that's worth adding is when it's a `d2l-table-col-sort-button` with a different sibling (e.g., the case with the col-sort-button next to the context menu).